scalameter-core/src/main/scala/org/scalameter/MeasureBuilder.scala

```diff
@@ -38,12 +43,12 @@ class MeasureBuilder[T, U](
   
       if (ctx(Key.exec.requireGC)) compat.Platform.collectGarbage()
   
-      val measurements = measurer.measure[T, U](
+      val measurements = measurer.measure[T](
         ctx, ctx(Key.exec.benchRuns),
         setup, teardown, regen, b
       )
   
-      resultFunction(measurements)
+      resultFunction(measurements.map(_.value))
```

Comment:
Yes, I've cleaned the project and removed `target/benchmarks`. 
I'm wondering if acceptable would be to make it that way:
- unwrap values and units  right after call to the `measure` (same as it's here)
- leave `Aggregator` as function from `Seq[T] => T`
- provide implicit function that lifts `Aggregator` to the `Seq[Quantity[T]] => Quantity[T]`
We wouldn't struggle with this weird error and users would get the possibility to both use aggregators and get value with units in the inline benchmarking.

What do you think about it?